base.go

```diff
@@ -89,7 +89,7 @@ func SetupRootfs(o BaseLayerOpts) error {
 	}
 }
 
-func importContainersImage(is *types.ImageSource, config types.StackerConfig, progress bool) error {
+func importContainersImage(is types.ImageSource, config types.StackerConfig, progress bool) error {
```

Comment:
it's probably safer this way then, but does this have a non-negligable performance impact due to full struct copy?

Reply:
It shouldn't, and like I say in the commit message, there is some go creedo somewhere that I read that says "don't use pointers unless you intend to mutate data". In fact it might be faster due to less dereferences...

Reply:
One dereference at function call cannot be slower than allocating and copying the struct to a new location :)

But I'll buy that it's the "proper" thing to do, so resolving this, thanks.

Reply:
> One dereference at function call cannot be slower than allocating and copying the struct to a new location :)

I don't agree, but I also don't think it matters.